docs/BUILD_AND_TEST.md

```diff
@@ -1,26 +1,54 @@
  
 # Build and Test
 ## Getting Started
+
 To build snap you'll need:
+
 * [Golang >= 1.4](https://golang.org)
```

Comment:
I don't know :) I installed 1.6. That's definitely outside of the scope of these changes.